src/components/Navigation/index.js

```diff
@@ -0,0 +1,64 @@
+import PropTypes from "prop-types";
+import React, { Component } from "react";
+
+import "./index.css";
+
+class Navigation extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { isActive: false };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    this.setState(prevState => ({
+      isActive: !prevState.isActive
+    }));
+  }
+
+  render() {
+    const { isRevealing } = this.props;
+    const { isActive } = this.state;
+    const navigationClassName = `Navigation ${
+      isRevealing ? "Navigation--revealing" : ""
+    }`;
+    const detailsClassName = `Navigation-details ${
+      isActive ? "Navigation-details--active" : ""
+    }`;
+    return (
+      <nav className={navigationClassName}>
+        {isRevealing && (
+          <button className="Navigation-command" onClick={this.handleClick}>
+            Menu
```

Comment:
From my understanding, this text is hidden and the hamburger icon replaces it via CSS. Is this accessible? Is the hamburger on its own understood enough to be a menu?

Reply:
> this text is hidden and the hamburger icon replaces it via CSS. Is this accessible?

It's hidden using `text-ident` so VoiceOver reads it.

> Is the hamburger on its own understood enough to be a menu?

That's unclear. There's a follow up issue to test this (amongst other aspects of the DS) with service users: https://github.com/barnardos/design-system/issues/95